static/openapi/prerendering-spec.json

```diff
@@ -285,14 +58,18 @@
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Success status"
+                      "description": "Success status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Success message"
+                      "description": "Success message."
                     }
                   },
-                  "required": ["status"]
+                  "required": ["status"],
+                  "example": {
+                    "status": "OK",
+                    "message": "Cache purged successfully"
```

Comment:
@joubert-macrometa, do we usually put periods at the end of messages?

Reply:
Done